components/tag-list/tag-list.js

```diff
@@ -349,7 +343,7 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 			});
 			this._chomp();
 			this.requestUpdate();
-		});
+		}, 40);
```

Comment:
I was getting incorrect layout/dimensions without this delay.  Unfortunately the longer this delay, the more noticeable the FOUC in Safari, though it was still noticeable before these changes.